server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1635,6 +1635,17 @@ private SplitRowSpec findSplitRow(Collection<FileRef> files) {
       return null;
     }
 
+    if (keys.isEmpty()) {
+      log.info("Cannot split tablet " + extent + ", files contain no data for tablet.");
+
+      // set the following to keep tablet from attempting to split until the tablets set of files
+      // changes.
+      sawBigRow = true;
+      timeOfLastMinCWhenBigFreakinRowWasSeen = lastMinorCompactionFinishTime;
```

Comment:
This could be another PR but might want to come up with better naming for these variables given they are being used for this case.

Reply:
I will do that as a follow on change.  Do not want them in the same commit.

Reply:
#1016